### PR TITLE
Add platform team to CODEOWNERS for src/lightning_app

### PR DESCRIPTION
**Pull request by rlizzo**
## What does this PR do?

added @Lightning-AI/lai-platform team to CODEOWNERS of src/lightning_app as per request of @narJH27 

cc @borda @carmocca @akihironitta @Borda 

Fixes: GRID-10479

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- ~[ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)~
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?~
- ~[ ] Did you make sure to **update the documentation** with your changes? (if necessary)~
- ~[ ] Did you write any **new necessary tests**? (not for typos and docs)~
- ~[ ] Did you verify new and **existing tests pass** locally with your changes?~
- ~[ ] Did you list all the **breaking changes** introduced by this pull request?~
- ~[ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)~

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

Yes I had fun coding!

### CODE REVIEW

File: .github/CODEOWNERS
Comment (rlizzo) [reply .github/CODEOWNERS:49]: correct. Let me re-adjust the settings here based on Jirka's feedback

Comment (rlizzo) [reply .github/CODEOWNERS:49]: addressed! thanks

File: .github/CODEOWNERS
Comment (Borda) [inline .github/CODEOWNERS:50]: I would prefer to have a specific person because as team it may easily happen someone not experienced would gain quite high access level... :rabbit:

**Review by Borda — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (Borda) [reply .github/CODEOWNERS:50]: > We want to spread the knowledge across our team as well so that folks are comfortable supporting this and helping future reviews as well

That is exactly what code-ownership is about, so when some developers become comfortable and responsible for changing some codebase, let's add them :)

**Comment by Borda** [comment]
@tchaton, could you please make a suggestion on who of the @Lightning-AI/lai-platform could be code owners for what part of the App codebase?

File: .github/CODEOWNERS
Comment (tchaton) [reply .github/CODEOWNERS:50]: Hey @narJH27 This is definitely not being merged 😁

The goal is to move toward stability and strong ownership. We won t give merge power to a Team, only indivivual with the right level of expertise.

**Comment by tchaton** [comment]
> @tchaton, could you please make a suggestion on who of the @Lightning-AI/lai-platform could be code owners for what part of the App codebase?

Yes, let me do that, this is a better approach.

File: .github/CODEOWNERS
Comment (narJH27) [reply .github/CODEOWNERS:50]: @tchaton @Borda I disagree with this decision to close this PR. We on platform are making developments towards convergence to lai and our development velocity is currently being hampered by not having a process in place where we're able to get reviews from the current codeowners of this repo fast enough. I've discussed this with @Borda in detail and we agreed to have the current members of the platform team as co-codeowners for the pieces outlined in this PR.

While this will certainly not resolve the issues with our development velocity entirely, it will certainly give us the ability to learn more about these areas of the codebase and help in reviewing PRs from other authors, and also help us unblock at least 1 review request without having to chase the current codeowners for the repo. 

I vote to re-open this PR and amend the change to reflect just the current members of the platform team as opposed to listing the entire team in here. @rlizzo will make this change once we're all aligned here. cc: @JatheenA

**Pull request closed without merging**
